services/frontend-service/Earthfile

```diff
@@ -67,8 +68,11 @@ node-modules:
 
 lint-ui:
     FROM +deps-ui
-    RUN pnpm eslint-check
-    RUN pnpm scss-check
+    RUN pnpm eslint-fix
+    RUN pnpm scss-fix
```

Comment:
why are we calling the "fix" targets here? I think in a lint-ui target we should call the lint targets, not the fix targets.

Reply:
rebase fumble, removed